modules/exploits/multi/http/mantisbt_manage_proj_page_rce.rb

```diff
@@ -0,0 +1,123 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Mantis v1.1.3 Remote Code Execution',
+      'Description'    => %q{
+        Mantis v1.1.3 and earlier is vulnerable to a post-authentication Remote Code Execution vulnerability in the sort parameter of the manage_proj_page.php page.
+      },
+      'Author'         => [
+        'EgiX',      # Exploit-DB Entry Author
+        'Lars Sorenson'      # MSF module author
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['EDB', '6768'],
+          ['URL', 'https://www.exploit-db.com/exploits/6768/'],
+        ],
+       'Privileged' => false,
+       'Platform'   => ['php'],
+       'Arch'       => ARCH_PHP,
+       'Targets' =>
+          [
+            [ 'Mantis <= 1.1.3', { } ],
+          ],
+      'DisclosureDate' => 'Oct 16, 2008',
```

Comment:
This will probably fail the tests.

Please ensure you run `./tools/dev/msftidy modules/exploits/multi/http/mantisbt_manage_proj_page_rce.rb`

Reply:
I did, and this didn't fail